pages/common/cmake.md

```diff
@@ -1,13 +1,28 @@
 # cmake
 
-> Cross-platform build system generator.
-> It generates Makefiles, Visual Studio projects or others, depending on the target system.
+> Cross-platform build automation system, that generates recipes for native build systems.
 > More information: <https://cmake.org/cmake/help/latest/manual/cmake.1.html>.
 
-- Generate a Makefile and use it to compile a project in the same directory as the source:
+- Generate a build recipe in the current directory with `CMakeLists.txt` from a project directory:
 
-`cmake && make`
+`cmake {{path/to/project_directory}}`
 
-- Generate a Makefile and use it to compile a project in a separate "build" directory (out-of-source build):
+- Generate a build recipe, build type setted to `Release` with CMake variable:
 
-`cmake -H. -B {{build}} && make -C {{build}}`
+`cmake {{path/to/project_directory}} -D CMAKE_BUILD_TYPE=Release`
```

Comment:
Token syntax on variable parameters, when reasonable, is preferable, even when the description explicitly states the value of such a parameter, as in this case.
```suggestion
`cmake {{path/to/project_directory}} -D {{CMAKE_BUILD_TYPE=Release}}`
```